README.md

```diff
@@ -1,10 +1,25 @@
-tessellation
-===========
+# tessellation
+
 
 ![build](https://img.shields.io/github/actions/workflow/status/Constellation-Labs/tessellation/release.yml?label=build)
 ![version](https://img.shields.io/github/v/release/Constellation-Labs/tessellation?sort=semver)
 
-## Running L0 & L1 in Kubernetes
+The Constellation Network Node Software, written in Scala, ready for Kybernetes Deployment.
+
+## Documentation
+
+You'll find thorough resources on the documentation site https://docs.constellationnetwork.io.
+
+Ensure to go through the [Constellation Network Primer](https://docs.constellationnetwork.io/learn) to review the main characteristics of Constellation.
+
+## Dev Setup / Contributing
+
+* [Dev System Setup](SETUP.md)
+* [Contributing Instructions and Guidelines](CONTRIBUTING.md)
+
+## Quick Start
+
+Running L0 & L1 in Kubernetes
```

Comment:
References to `L0` and `L1` may be ambiguous. Are you referring to the _Global_ L0 and L1, or _Metagraph_ ?

Reply:
This is original text.

Reply:
squashed & forced pushed, including a small correction for this.